### PR TITLE
network: update CMDUncknown

### LINKED ISSUE #1135: Change CMDUnknown to CMDNotFound and implement its handling
It should be emitted for missing elements of `inv` request.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Closes #1135

Post-preview2.

### CODE REVIEW

File: pkg/network/server.go [hunk 1]
```diff
@@ -571,6 +576,9 @@ func (s *Server) handleGetDataCmd(p Peer, inv *payload.Inventory) error {
 			}
 		}
 	}
+	for _, invGroup := range payload.CreateGroup(inv.Type, notFound) {
```
Comment (roman-khimov) [inline pkg/network/server.go:579]: You don't really need it here, GetData is limited with `MaxHashesCount`, so the number of missing hashes can't be more than that. But it'd be nice to enforce that restriction in `Inventory` payload decoder.

**Comment by roman-khimov** [comment]
But what about handling this command? Our node will receive it from its peers.

**Comment by AnnaShaleva** [comment]
> But what about handling this command? Our node will receive it from its peers.

Well, it's a bit strange, but we shouldn't handle this command (see https://github.com/neo-project/neo/blob/master/src/neo/Network/P2P/RemoteNode.ProtocolHandler.cs#L116). Am I missing something?

**Comment by roman-khimov** [comment]
I've just checked, we're ignoring unknown commands by default and it's enough for now.

**Pull request merged**